build.sbt

```diff
@@ -0,0 +1,28 @@
+val appName = "address-lookup-performance-tests"
+val appVersion = "0.1.0-SNAPSHOT"
+
+lazy val root = (project in file("."))
+  .enablePlugins(GatlingPlugin)
+  .enablePlugins(CorePlugin)
+  .enablePlugins(JvmPlugin)
+  .enablePlugins(IvyPlugin)
+  .enablePlugins(SbtAutoBuildPlugin)
+  .settings(
+    organization := "uk.gov.hmrc",
+    name := appName,
+    version := appVersion,
+    scalaVersion := "2.12.12",
+    scalacOptions ++= Seq("-feature"),
+    retrieveManaged := true,
+    initialCommands in console := "import uk.gov.hmrc._",
+    parallelExecution in Test := false,
+    publishArtifact in Test := true,
```

Comment:
Do we have the utility of lines 16, 18 and 19 documented anywhere?  removing them seems to have no effect so I assume the above are defaults.